### PR TITLE
issue #29043 Have All Sites as an option in the dropdown

### PR DESCRIPTION
**Pull request by ChristopherCotnoir**

### CODE REVIEW

**Comment by ChristopherCotnoir** [comment]
Requires xtuple/xtuple#2919.

**Comment by gilmoskowitz** [comment]
This addresses the bug as reported but does not handle the general case. Consider the ramifications of changing the parameter widget instead.

**Comment by gilmoskowitz** [comment]
On second thought, this isn't a bug at all. If all sites are desired then the user should remove sites from the list of filters. That's what the - button is for.

**Comment by ChristopherCotnoir** [comment]
The All Sites filter I added shows item sources with access to all sites, but NOT ones with access to only one site, so it does still function as a filter. Whether this filter is still needed, and whether filtering on one site should show sites with access to all sites (currently it doesn't) is up for discussion.

File: guiclient/displays/dspItemSources.cpp [hunk 1]
```diff
@@ -35,9 +35,16 @@ dspItemSources::dspItemSources(QWidget* parent, const char*, Qt::WindowFlags fl)
 //  setNewVisible(true);
 //  setQueryOnStartEnabled(true);
 
+  QString qrySite = "SELECT 0, 'All Sites' AS name, '' AS code "
```
Comment (gilmoskowitz) [inline guiclient/displays/dspItemSources.cpp]: Given prior comments on this issue and discussion with Support, let's change the text from `All Sites` to `[ Not Site Restricted ]`. We also need to make this string translatable.

The rationale is that `All Sites` in all(?) other filters in the application means "Show all rows regardless of the site/warehous_id;" here it means explicitly "Show all rows with a certain value, where that certain value means 'all sites/warehouses'."

**Pull request merged**